libs/infinity_emb/infinity_emb/transformer/vision/utils.py

```diff
@@ -3,24 +3,58 @@
 
 from infinity_emb._optional_imports import CHECK_PIL, CHECK_REQUESTS
 from infinity_emb.primitives import ImageCorruption, ImageSingle
+from typing import Union, List, Sequence
 
 if CHECK_PIL.is_available:
     from PIL import Image  # type: ignore
+    from infinity_emb.primitives import ImageClassType
```

Comment:
This is incorrect and should be put outside if statement. Otherwise typing will fail, when PIL is not installed.

Reply:
Yes, if not installed, type hint won't pass. Just fix this.